packages/config/src/lib/createPlugins.ts

```diff
@@ -68,6 +83,31 @@ export const createPlugins = ({
     ...(hasTsConfig ? [new ForkTsCheckerWebpackPlugin()] : []),
     ...(plugins || []),
     ...(internalHotReload ? [new ReactRefreshWebpackPlugin()] : []),
+    // Put the sentry plugin at the end of the plugins array
+    ...(process.env.ENABLE_SENTRY && initSentry
```

Comment:
What is the initSentry for? I think we can also leave that out and have no sentry sepcific config within the fec.config.

I'm not sure if it's still needed, but we need to pass through the environment variable in the `fec` script(s), as it's running in a sub-shell that might not have those vars.